modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/Loader.scala

```diff
@@ -202,4 +206,8 @@ object Loader {
         Monitoring[F].alert(Monitoring.AlertPayload.error(error.toString)) *>
         Monitoring[F].trackException(error)
   }
+
+  private def initRetryLog[F[_]: Logging: Applicative](e: Throwable, d: RetryDetails): F[Unit] =
+    Logging[F].info(show"Exception from init block. $d") *>
+      Logging[F].debug(show"Caught exception from init block: ${e.toString}")
```

Comment:
This commit looks good.  It will stop the app forever crashing, which was the main concern.

My concern though, is that now we will never find out if the password is wrong.  You log a message, which is a good start, but realistically nobody will see the log message.  Is it possible to send an AlertPayload to the webhook so that we get a notification when there is a problem?

I am checking with techops to confirm they want an alert.

Reply:
I think that is a good idea. Let's wait what TechOps will say on this.